apps/sertra/components/LaunchingModManPopup.module.css

```diff
@@ -0,0 +1,67 @@
+.background {
+  display: flex;
+  position: fixed;
+  left: 0;
+  top: 0;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  background: #00000060;
+}
+
+.popUp {
+  background: #242e48;
+  border-radius: 0.5rem;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 1.53125rem;
```

Comment:
I rounded those down a bit. I just added all the decimals to make sure I got it right style wise. Though this might be something I need to mention to @gkurck , so that he can make sure none of the dimensions are non by 8 divisible.

Reply:
Why do you need gkurck for that, don't you have a calculator 😅 And why would the margin need to be divisible by 8? Is that some sort of stated design goal?

Reply:
The "divisible by 8" thing comes from the fact that the since the browser base pixel value is (in most cases) 16px, the measurements in the designs follow that pattern. so 1rem = 16px, .5rem = 8px and so on. This also helps to make sure the units are always divisible - compare that to base of 10, where a third would be 3.333333, which isn't ideal.

That being said, the amount of decimals here are an overkill, and you should be able to survive with divisions of 1.

Reply:
So yeah, it's not 8 but rather 16. And "divisible by 16" is incorrect too since 8 is not divisible by 16, but the other way around. (Not 100% sure but I'd think that's how the semantics go.)